otherfile/font_wizard.py

```diff
@@ -100,7 +107,14 @@ def is_ascii(s):
     if len(font_dic) == 95:
         print("Done!")
         print("Font dictionary : \n")
-        print(font_dic)
-        print("\nThis font is {0}".format(ascii_flag))
+        print("{}_dic = ".format(font_name),font_dic)
+        print("- Add this dictionary to the end of text_dic3.py")
+        print("- This font is {0} : ".format(ascii_flag))
```

Comment:
Considering next line if, is this line necessary?

Reply:
Why not?
I think it would be better if we show to user that this font is ASCII or not 💯